gp-templates/helper-functions.php

```diff
@@ -366,12 +369,17 @@ function textareas( $entry, $permissions, $index = 0 ) {
 
 function display_status( $status ) {
 	$status_labels = array(
-		'current'  => _x( 'current', 'Single Status', 'glotpress' ),
-		'waiting'  => _x( 'waiting', 'Single Status', 'glotpress' ),
-		'fuzzy'    => _x( 'fuzzy', 'Single Status', 'glotpress' ),
-		'old'      => _x( 'old', 'Single Status', 'glotpress' ),
-		'rejected' => _x( 'rejected', 'Single Status', 'glotpress' ),
+		'current'           => _x( 'current', 'Single Status', 'glotpress' ),
+		'waiting'           => _x( 'waiting', 'Single Status', 'glotpress' ),
+		'fuzzy'             => _x( 'fuzzy', 'Single Status', 'glotpress' ),
+		'old'               => _x( 'old', 'Single Status', 'glotpress' ),
+		'rejected'          => _x( 'rejected', 'Single Status', 'glotpress' ),
+		'changes_requested' => _x( 'Changes requested', 'Single Status', 'glotpress' ),
```

Comment:
All the other statuses start with lower case. This one should too, I think.

Reply:
Updated here https://github.com/GlotPress/GlotPress/pull/1451/commits/025e9330e97181bb1b8aef3e272a43416ca6ff23.